prime-router/src/main/kotlin/cli/tests/TestReportStream.kt

```diff
@@ -841,6 +845,7 @@ class Strac : CoolTest() {
     override val status = TestStatus.SMOKE
 
     override fun run(environment: ReportStreamEnv, options: CoolTestOptions): Boolean {
+        initListOfGoodReceiversAndCounties(environment)
         ugly("Starting bigly strac Test: sending Strac data to all of these receivers: $allGoodCounties!")
         var passed = true
```

Comment:
See above about assuming success; this seems like a broader issue - can you file a GitHub issue to revisit this?

Reply:
ok for now as test code.